sound/soc/sof/amd/acp.c

```diff
@@ -235,6 +251,7 @@ int configure_and_run_sha_dma(struct acp_dev_data *adata, void *image_addr,
 		}
 	}
 
+	snd_sof_dsp_write(sdev, ACP_DSP_BAR, ACP_SHA_DMA_INCLUDE_HDR, ACP_SHA_HEADER);
```

Comment:
This is required for Signed Firmware only. It will break the Unsigned fw loading. Please add condition check

Reply:
Okay agree, Thanks
I will add condition check.